util/transform.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+
+// util
+const generateKeywords = require('./keywords');
+
+const PHRASE_PREFIX = 'botto-kun';
+const COMMAND_PREFIX = '!';
+const ALL_KEYWORDS = generateKeywords();
+
+
+const transformMessage = (message) => {
+  // botto-kun message format
+  const bkmf = {
+    rawMessage: message,
+  };
+
+  // lower case it and check prefixes
+  const lcMessage = message.content.toLowerCase();
+  bkmf.isPhrase = lcMessage.indexOf(PHRASE_PREFIX) === 0;
+  bkmf.isCommand = lcMessage.indexOf(COMMAND_PREFIX) === 0;
+
+  let cleanMessage = null;
+  if (bkmf.isPhrase) { cleanMessage = lcMessage.replace(/[^a-z0-9 ]/g, '').slice(PHRASE_PREFIX.length); }
```

Comment:
not a huge deal, but all of these would be much easier to read with some line breaks

Reply:
refactored